### ISSUE TITLE
A few missing features for Trajectories Addon

### ISSUE THREAD

**Issue by Dunbaratu** [KSP-KOS/KOS#2263]
Here's a few features I noticed were missing from the Trajectories Addon when I was testing PR #2220.
These shouldn't delay merging PR #2220 because that PR is desperately needed to make the Trajectories Addon work *at all*.  But they shouldn't be forgotten either, so I'm making this a separate issue to correct after PR #2220 is merged:

- Once ``Addons:TR:SETTARGET()`` has been called, there's no way to actually un-set the target from the script.  (Also no way for the script to un-set it after the user manually set it).
- A script cannot query *what* the Trajectories target is currently set to.  It can query that the target exists, but not what it is.  There should be a way to obtain the GEOPOSITION from it.
- The manual GUI interface lets a user pick a descent profile that is neither prograde nor retrograde, but something in between, with the sliders.  There is no way for a script to do this right now.  Perhaps making a script able to set/get the value of the 4 variables attached to those slider values would help.

**Comment by PiezPiedPy** [comment]
@Dunbaratu Thanks for the heads up, I'll add a method to extract the Target position along with a Clear Target method.
Also going to add methods to set and retrieve the Descent Profile nodes.

**Comment by rlangeoi** [comment]
Since PR #2220 has been merged, shouldn't Trajectories 2.0 compatibility be available in the latest v1.1.5.2 release for 1.4.1 (at least with basic support) ? It doesn't seem to be working for me and since TR < 2.0 doesn't exist for 1.4 I can't downgrade it

**Comment by austinjames314** [comment]
Is there any way to get the existing functionality into a game now, in an as-is state? I don't care about anything other than getting the coordinates of the impact point, so I don't need any of the functions about the current Trajectories target.

**Comment by PiezPiedPy** [comment]
Due to the amount of requests on the Trajectories forum thread I made a build of kOS and posted a link to this zip https://www.dropbox.com/s/ykvqsqlgc8714h3/kOS_develop.zip?dl=0

**Comment by PiezPiedPy** [comment]
New API methods where added for the Descent Profile for Trajectories v2.3.2 `ResetDescentProfile DescentProfileAngles DescentProfileModes`.

The next release of Trajectories being v2.4.0 will have new API methods `GetTarget` and `ClearTarget`.

I plan on adding them to kOS in the near future 😉

**Comment by PiezPiedPy** [comment]
@Dunbaratu or whoever can help 😉 
So I've basically got the Trajectories v2.4.0 API added to kOS and all is fine except some of the setters for a few properties are giving me an error in the kOS terminal, since I'm rubbish on how kOS does it's thing I need help resolving the issue.

Terminal output:
```
set addons:tr:descentprofilemodes to List(true,false,true,false).
Object must implement IConvertible.
At interpreter, line 28
set addons:tr:descentprofilemodes to List(true,false,true,false).
```

corresponding commit on my fork is at https://github.com/PiezPiedPy/KOS/commit/a2e590e3ecd489737a8e7ae6fe8572c924eb2c3c

**Comment by PiezPiedPy** [comment]
Finally got this implemented so I suppose this issue can be closed.

**Comment by Dunbaratu** [comment]
Closed by #2747
